docs/root/start/quick-start/configuration-dynamic-filesystem.rst

```diff
@@ -8,6 +8,15 @@ protocol.
 
 When the files are changed on the filesystem, Envoy will automatically update its configuration.
 
+.. seealso::
+
+   :ref:`Atomic swap <_config_runtime_symbolic_link_swap>`
```

Comment:
Please remove the prefix `_` in the link.

Reply:
I will try.

Reply:
@wbpcode thanks, it helps.